framework/src/controller/validator/keywords/env/index.js

```diff
@@ -18,7 +18,23 @@ const compile = (schema, parentSchema) => {
 			  };
 
 	return function(data, dataPath, object, key) {
-		const variableValue = process.env[envVariable.name];
+		let variableValue = process.env[envVariable.name];
```

Comment:
```suggestion
		const variableValue = process.env[envVariable.name];
```

Reply:
I don't think that's possible because he is mutating the value in the lines below.

Reply:
oops, sorry

Reply:
Must stay `let` here as I modify the `variableValue` if it needs to be parsed.